bin/p2-install-hook/main.go

```diff
@@ -49,11 +50,19 @@ func main() {
 
 	if *isPreparerManifest {
 		config := manifest.GetConfig()
```

Comment:
Shouldn't this line be using the alias of `p2manifest.GetConfig()` for clarity?

Reply:
no, manifest is not an alias, it's the real manifest we are using